client/admin/src/components/Table.vue

```diff
@@ -121,5 +126,17 @@ export default class Table extends Vue {
   public resetErrors(): void {
     this.resetPaginatorInfoError();
   }
+
+  public checkIfLink(element: string) {
```

Comment:
@AlexeiaW how strict would this check need to be?

I think we're going to need a validation lib (Yup) to handle validation in general. Yup is a great flexible option. If this needs a strict url check, we could use the url method: https://github.com/jquense/yup#stringurlmessage-string--function-schema

Reply:
@luke-siedle this was an older function, but not used anymore. Not sure why it still showed up.

Good to know about Yup, will use it when needed next